### DISCUSSION TITLE
using directus as auth ?

### DISCUSSION THREAD

**Discussion by imknight** [directus/directus#13346] [Archive]
hi, new user here, I have a question on the auth part. I just wonder can I use Directus as a core system for the backend and the user can use the Directus auth to use other systems? something like the system that implements google login where using google auth for users to access their system ( using Directus as the google login for this example )

**Comment by xu4wang** [comment]
Cannot by default, since Directus only support OAuth client protocol.

but I think it's possible by implement OAuth server side via a plugin.
